CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to PyCon India 2018 website. Your valuable contribution is much appreciated.
+
+The following is a set of guidelines for contributing to PyCon India 2018.
+
+## How Can I Help?
+
+Start looking through the issues and watch out for any issues with the `bug` or `help wanted` labels. Issues marked with these labels haven't been assigned to anyone yet and we would love for you to work on them.
+
+### Request a feature
```

Comment:
### make it a separate heading (H3 to be exact) since you have used ## (H2) everywhere else. It would be great if you followed the same for uniformity.

Reply:
made to H2